setup.py

```diff
@@ -19,6 +21,8 @@
     'psutil==2.1.3',
     'hypothesis',
     'pgpy',
+    'tox==2.3.1',
+    'coverage==4.0.3',
```

Comment:
these are the test_require, which we use devDep.py to install using pip... they are not installed for anyone who isn't running tests.